ansible/roles/ocp4-workload-open-data-hub-olm/readme.adoc

```diff
@@ -0,0 +1,135 @@
+= ocp4-workload-open-data-hub-olm - Deploy Open Data Hub using the community operator included in OLM
+
+== Role overview
+
+* This role deploys Open Data Hub. It consists of the following playbooks:
+** Playbook: link:./tasks/pre_workload.yml[pre_workload.yml] - Sets up an
```

Comment:
This is a tasks file, not a playbook.

Reply:
I don't mind changing this, but I copy/pasted it from other roles which all use the same boilerplate text. For example, the ocp4_workload_piplines, ocp4_workload_logging, and ocp4_workload_serverless roles that wkulhanek suggested I follow all include this stanza, along with all of the others I randomly checked.

I'll change it if you're going to block the PR because of it, but personally, I'm more for consistency and then to address this globally for all of our readmes.